data/json/items/fluff.json

```diff
@@ -131,6 +516,166 @@
     "flags": [ "NO_REPAIR" ],
     "use_action": [ "PLAY_GAME" ]
   },
+  {
+    "id": "tangram",
+    "type": "GENERIC",
+    "category": "other",
+    "name": { "str": "tangram set" },
+    "description": "A modest wooden box containing a square base, two small triangles, two large triangles, a parallelogram, a rhomboid, and a semi-circle, all painted in different colors to play the traditional Chinese puzzle of tangram.  At the box, there are several figures that you can make with the pieces.",
+    "weight": "788 g",
+    "volume": "1500 ml",
+    "price": 2000,
+    "price_postapoc": 500,
+    "material": [ "wood" ],
+    "symbol": "?",
+    "color": "brown",
+    "flags": [ "NO_REPAIR" ],
+    "use_action": [ "PLAY_GAME" ]
+  },
+  {
+    "id": "chinese_checkers",
+    "type": "GENERIC",
+    "category": "other",
+    "name": { "str": "Chinese checkers set" },
+    "description": "A wooden box containing a wooden board with a star-shaped pattern, ten holes in each of its six arms, and a set of round pegs used to play Chinese checkers.  At the box, there are a comprehensible explanation of how to play the game.",
+    "weight": "1134 g",
+    "volume": "750 ml",
+    "price": 2000,
+    "price_postapoc": 500,
+    "material": [ "wood" ],
+    "symbol": "?",
+    "color": "brown",
+    "flags": [ "NO_REPAIR" ],
+    "use_action": [ "PLAY_GAME" ]
+  },
+  {
+    "id": "mancala",
+    "type": "GENERIC",
+    "category": "other",
+    "name": { "str": "mancala set" },
+    "description": "A somewhat big wooden box containing a rectangular wooden board with a series of small pits, as well as a number of game pieces that looks like different coloured beads.  The board features two rows of pits, with each player controlling the row of pits closest to them.  The game pieces are distributed evenly among the pits at the beginning of the game, and players take turns picking up and redistributing the pieces according to the specific rules.  At the box, there are a comprehensible step-by-step of how to play mancala.",
+    "weight": "1361 g",
+    "volume": "1000 ml",
+    "price": 2000,
+    "price_postapoc": 500,
+    "material": [ "wood" ],
+    "symbol": "?",
+    "color": "brown",
+    "flags": [ "NO_REPAIR" ],
+    "use_action": [ "PLAY_GAME" ]
+  },
+  {
+    "id": "connect_four",
+    "type": "GENERIC",
+    "category": "other",
+    "name": { "str": "Four in a Row set" },
+    "description": "A board game cardboard box containing a plastic grid with seven columns and six rows, and 42 coloured discs, alongside a small booklet explaining how to play Four in a Row.",
```

Comment:
I added the term "board game cardboard box" because the cardboard boxes used to store board games are pretty much unique and specific, I see no harm in the term.